osh/hub/scripts/osh-retention

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env python3
+# SPDX-License-Identifier: GPL-3.0-or-later
+# SPDX-FileCopyrightText: Copyright contributors to the OpenScanHub project.
+
+"""
+Script for cron that performs the retention policy
+"""
+import datetime
+import logging
+import os
+import shutil
+from datetime import date
+
+logger = logging.getLogger(__name__)
+
+os.environ['DJANGO_SETTINGS_MODULE'] = 'osh.hub.settings'
+
+
+def delete_tasks_results(tasks_to_delete):
+    from kobo.hub import models
+    for task in tasks_to_delete:
+        try:
+            path_to_delete = models.Task.get_task_dir(task.id)
+            shutil.rmtree(path_to_delete)
+        except OSError as e:
+            logger.error("Error deleting tasks folder: %s", e)
+
+
+# In order to identify personal tasks, we are using the owner field. If the owner doesn't contain any / or @, we assume
+# that it is a personal task.
+def identify_personal_task(owner):
+    # We could use regexps here if we want to make this more complex but for this specific case
+    # built-in string methods are enough, and we are more perfomance-oriented.
+    return "/" not in owner and "@" not in owner
+
+
+def main():
+    import django
+    django.setup()
+
+    # Retention policy Task
+    logger.info("Starting Retention Policy Enforcement")
+
+    from osh.hub.scan import models
+    tasks = models.Task.objects.all()
```

Comment:
On a second thought, we definitely want to apply this policy only to finished tasks:
```python
from kobo.client.constants import FINISHED_STATES
tasks = models.Task.objects.filter(state__in=FINISHED_STATES)
```